<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilerForm</class>
 <widget class="QWidget" name="TilerForm">
  <property name="windowTitle">
   <string>VirtuGhan Tiler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <item>
    <widget class="QGroupBox" name="groupBoxConn">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridConn">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBackend">
        <property name="text">
         <string>Backend URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="backendUrlLine">
        <property name="placeholderText">
         <string>http://127.0.0.1:8000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLayerName">
        <property name="text">
         <string>Layer Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="layerNameLine">
        <property name="placeholderText">
         <string>VirtuGhan Tiler</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBoxParams">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridParams">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStart">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="startDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelEnd">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="endDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="labelCloud">
        <property name="text">
         <string>Cloud cover (%)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="cloudSpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="labelBand1">
        <property name="text">
         <string>Band 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="band1Combo"/>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="labelBand2">
        <property name="text">
         <string>Band 2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="band2Combo"/>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="labelFormula">
        <property name="text">
         <string>Formula</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="formulaLine">
        <property name="placeholderText">
         <string>(band2 - band1) / (band2 + band1)</string>
        </property>
       </widget>
      </item>

      <item row="6" column="0">
       <widget class="QLabel" name="labelTimeseries">
        <property name="text">
         <string>Time series (aggregate)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="timeseriesCheck">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>

      <item row="7" column="0">
       <widget class="QLabel" name="labelOp">
        <property name="text">
         <string>Aggregation</string>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="operationCombo">
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonRow">
     <item>
      <widget class="QPushButton" name="addLayerBtn">
       <property name="text">
        <string>Add XYZ Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpBtn">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
